unit-tests/src/org/commcare/dalvik/application/CommCareTestApp.java

```diff
@@ -0,0 +1,24 @@
+package org.commcare.dalvik.application;
+
+import org.commcare.android.database.UnencryptedHybridFileBackedSqlStorage;
+import org.commcare.android.database.UnencryptedHybridFileBackedSqlStorageMock;
+import org.javarosa.core.services.storage.Persistable;
+
+/**
```

Comment:
Left one that is sort of vague, would appreciate suggestions for ways to make it clearer.